Microsoft.Toolkit.Uwp.SampleApp/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.NETCore.UniversalWindowsPlatform": "5.1.0",
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
```

Comment:
Not sure about the crash, but there's no reason not to use the latest UWP tooling dependency.